NAMESPACE

```diff
@@ -1,5 +1,10 @@
 # Generated by roxygen2: do not edit by hand
 
+S3method("[",JoinKeys)
+S3method("[<-",JoinKeys)
+S3method("get_join_keys<-",JoinKeys)
+S3method("get_join_keys<-",TealData)
```

Comment:
We don't need this method for old class. Class will be removed soon
```suggestion
```

Reply:
Sure thing! Removed those